docs/development.md

```diff
@@ -157,3 +157,11 @@ for more docs.
 #### `vm-console-proxy` operand
 
 Installs the VM console proxy found in `data/vm-console-proxy-bundle`. A PR for updating the bundle is automatically created when a new version of the [vm-console-proxy](https://github.com/kubevirt/vm-console-proxy) is released.
+
+#### `tekton-pipelines` operand
+
+Installs the Tekton Pipelines found in `data/tekton-pipelines`. A PR for updating the bundle is automatically created when a new version of the [kubevirt-tekton-tasks](https://github.com/kubevirt/kubevirt-tekton-tasks) is released.
```

Comment:
There is no PR for updating pipelines. The PR is only for tekton tasks.
Can be fixed in a follow up PRs

Reply:
Ah okay. The pipelines bundle yaml files will be updated manually if needed?

Reply:
yes